src/utils/assignable.js

```diff
@@ -3,43 +3,97 @@
 // Do not use any browser or node-specific API!
 // -------------------------------------------------------------
 
+import { ActionInvalidObjectPropertyError } from '../shared/errors';
+
+const COMMAND_NAME_SUFFIX = 'Command';
+const arrayIndexOf        = Array.prototype.indexOf;
+const arrayMap            = Array.prototype.map;
+const arrayFilter         = Array.prototype.filter;
+const arrayConcat         = Array.prototype.concat;
+const objectToString      = Object.prototype.toString;
+const stringIndexOf       = String.prototype.indexOf;
+const stringEndsWith      = String.prototype.endsWith
```

Comment:
Move to the separate `src/utils/string-ends-with.js` file and add a comment
```js
// -------------------------------------------------------------
// WARNING: this file is used by both the client and the server.
// Do not use any browser or node-specific API!
// -------------------------------------------------------------
```

at the top of this file.